.github/workflows/on-push-to-main.yml

```diff
@@ -0,0 +1,31 @@
+# This workflow uses actions that are not certified by GitHub.
```

Comment:
just curious where this paragraph is coming from

Reply:
no idea lol, I'll remove that paragraph at the top of both